plugin.js

```diff
@@ -48,7 +48,7 @@ function onReqHandlerWrapper (fastify) {
 }
 
 function plugin (fastify, options, next) {
-  const secret = options ? options.secret || '' : ''
+  const secret = options.secret || ''
```

Comment:
For anyone else wondering: Fastify passes in an empty object by default for the options parameter.

Reply:
correct